cloud/shared/variable_definitions.json

```diff
@@ -4,5 +4,80 @@
     "secret": false,
     "type": "enum",
     "values": ["gcp", "azure", "aws"]
+  },
+  "EMAIL_SENDER": {
+    "required": true,
+    "secret": false,
+    "type": "enum",
+  "values": ["ses"]  
+  },
+  "ADMIN_AUTH_PROVIDER": {
+    "required": true,
+    "secret": false,
+    "type": "enum",
+  "values": ["activeDirectory"]  
+  },
+  "ADMIN_AUTH_FORMAT": {
+    "required": true,
+    "secret": false,
+    "type": "enum",
+  "values": ["OIDC"]  
+  },
+  "RESIDENT_AUTH_PROVIDER": {
+    "required": true,
+    "secret": false,
+    "type": "enum",
+  "values": ["openRadius", "idcs"]  
+  },
+  "RESIDENT_AUTH_FORMAT": {
+    "required": true,
+    "secret": false,
+    "type": "enum",
+  "values": ["OIDC", "SAML"]  
+  },
+  "application_name": {
+    "required": true,
+    "type": "tfvar",
+    "secret": "false"
+  },
+  "docker_repository_name": {
+    "required": true,
+    "type": "tfvar",
+    "secret": "false"
+  },
+  "docker_username": {
+    "required": true,
+    "type": "tfvar",
+    "secret": "false"
+  },
+  "docker_tag": {
+    "required": true,
+    "type": "tfvar",
+    "secret": "false"
+  },
```

Comment:
I'm thinking about taking this out of terraform and managing it through the CLI for the deploy script. but for now we can leave it as a tfvar